docs/index.asciidoc

```diff
@@ -380,12 +377,13 @@ URL to use
 How long to wait before checking if the connection is stale before executing a request on a connection using keepalive.
 You may want to set this lower, possibly to 0 if you get connection errors regularly
 Quoting the Apache commons docs (this client is based Apache Commmons):
-'Defines period of inactivity in milliseconds after which persistent connections must be re-validated prior to being leased to the consumer. Non-positive value passed to this method disables connection validation. This check helps detect connections that have become stale (half-closed) while kept inactive in the pool.'
-See https://hc.apache.org/httpcomponents-client-ga/httpclient/apidocs/org/apache/http/impl/conn/PoolingHttpClientConnectionManager.html#setValidateAfterInactivity(int)[these docs for more info]
+'Defines period of inactivity in milliseconds after which persistent connections must be re-validated prior to being leased to the consumer. Non-positive value passed to this method disables connection validation. 
+This check helps detect connections that have become stale (half-closed) while kept inactive in the pool.'
+See the https://hc.apache.org/httpcomponents-client-5.0.x/[apache.org docs] for more info.
```

Comment:
The link here is a tricky one to get right - this amended link points to the top level docs for the http client rather than the specific entry that was initially linked to. The version is also later than the version we actually use - the library that we are referring to here is pulled in by the manticore gem, which is pulled in by the logstash-mixin-http_client gem...

The current,correct active link is https://hc.apache.org/httpcomponents-client-4.5.x/current/httpclient/apidocs/org/apache/http/impl/conn/PoolingHttpClientConnectionManager.html#getValidateAfterInactivity()